supervision/tracker/byte_tracker/core.py

```diff
@@ -269,28 +270,39 @@ def callback(frame: np.ndarray, index: int) -> np.ndarray:
             )
             ```
         """
+        # Tensors is in the same order as detections
+        tensors = detections2boxes(detections=detections)
+
+        tracks = self.update_with_tensors(tensors=tensors)
+
+        final_detections = Detections.empty()
 
-        tracks = self.update_with_tensors(
-            tensors=detections2boxes(detections=detections)
-        )
-        detections = Detections.empty()
         if len(tracks) > 0:
-            detections.xyxy = np.array(
-                [track.tlbr for track in tracks], dtype=np.float32
-            )
-            detections.class_id = np.array(
-                [int(t.class_ids) for t in tracks], dtype=int
-            )
-            detections.tracker_id = np.array(
-                [int(t.track_id) for t in tracks], dtype=int
-            )
-            detections.confidence = np.array(
-                [t.score for t in tracks], dtype=np.float32
-            )
+            det_tlbrs = np.asarray([det[:4] for det in tensors])
```

Comment:
Let's not use abbreviations. They make the code harder to read. To be honest I don't know what `det_tlbrs` or `track_tlbrs` means :)